src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -83,6 +116,33 @@ public void setNumberOfChannels(String mediaInfoValue) {
 	}
 
 	/**
+	 * Get bits per sample for this audio track.
+	 * @return bits per sample (default is 16)
+	 */
+	public int getBitsperSample() {
+		return bitsperSample;
+	}
+
+	/**
+	 * Set bits per sample for this audio track.
+	 * @param bitsperSample bits per sample to set
+	 */
+	public void setBitsperSample(int bitsperSample) {
+		if (bitsperSample < 1) {
+			throw new IllegalArgumentException("Bits per sample can't be less than 1.");
+		}
+		this.bitsperSample = bitsperSample;
+	}
+
+	/**
+	 * Set bits per sample for this audio track with libmediainfo value
+	 * @param mediaInfoValue libmediainfo "BitDepth" value to parse
```

Comment:
@Sami32 This is really "nothing", but it's customary to have a blank line ```	 *``` between the descriptive text ```Set bits per sample for this audio track with libmediainfo value``` and the first ```@param```. It's also smart to end the sentence with a period ```.``` or CheckStyle will complain.